### PR TITLE
Release cascading_ext to Sonatype instead

### PR DESCRIPTION
**Pull request by joshk0**
Repeated process from LiveRamp/daemon_lib#26

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,6 +18,21 @@
...
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/LiveRamp/daemon_lib/issues</url>
+  </issueManagement>
+
+  <url>https://liveramp.com</url>
+
+  <licenses>
+    <license>
+      <name>MIT</name>
```
Comment (bpodgursky) [inline pom.xml]: I don't care one way or the other, but we should stay consistent on the licensing -- LICENSE.txt here is Apache 2.0

Comment (bpodgursky) [reply pom.xml]: Actually, doing some brief research, I think the patent troll protections in Apache 2.0 seem nice.  Might be worth bringing up with the arch board?  I am leaning Apache though.

Comment (joshk0) [reply pom.xml]: I'll stick with what's checked in for now. Thanks for the heads up

**Comment by joshk0** [comment]
@bpodgursky ping

File: README.md [hunk 2]
```diff
@@ -152,20 +152,11 @@ The tails of the MultiCombiner assembly will give access to the results for each
...
-  <id>repository.liveramp.com</id>
-  <name>liveramp-repositories</name>
-  <url>http://repository.liveramp.com/artifactory/liveramp-repositories</url>
-</repository>
-```
+## Download
 
-Version 0.1 is built off of Cloudera Hadoop 3, (CDH3u3).  The current snapshot version (1.6) is built against CDH4.1.2.  Both are available via Maven:
+You can either build cascading_ext from source as described below, or pull the
+latest jar from Maven Central. Version 0.1 is built off of Cloudera Hadoop 3,
```
Comment (bpodgursky) [inline README.md]: Is this true?  I don't see it deployed there yet http://repo1.maven.org/maven2/com/liveramp/

Comment (joshk0) [reply README.md]: no, you are right, it's not there yet.
What do you think about downgrading all this to snapshot style releasing? In the future I want to make another pass here to actually release things. I can just suppress the doc guidance for now.

Comment (bpodgursky) [reply README.md]: Yeah I'm fine just having snapshots for now, I just want to make sure we don't tell people it's available in central before it is.

**Comment by joshk0** [comment]
PTAL @bpodgursky

**Comment by bpodgursky** [comment]
:shipit:

**Pull request merged**